package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "checkout-ui-tests",
-  "version": "0.2.8",
+  "version": "0.2.10",
```

Comment:
Any reason for skipping `0.2.9`?

Reply:
Actually, I created the [`0.2.9`](https://github.com/vtex/checkout-ui-tests/pull/73/commits/8f34fef7b54fcab34cf1d2fdd71698c5f5d7de0c) too.